### PR TITLE
added getLanguages method to controller. It returns the broadest set …

### PR DESCRIPTION
**Pull request by amiklosi**
…of languages that are supported by all pies of controller.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ export default class PieClientSideController{
       });
   }
 
+  getLanguages() {
+    return _.intersection.apply(this, _.map(this._pies, (pie) => { return pie.pie.langs }));
```
Comment (edeustace) [inline lib/index.js:25]: a slightly more lodashy way? `_(this._pies).map('pie.langs').flatten().intersection().value()`

File: test/index-test.js [hunk 2]
```diff
@@ -35,24 +47,36 @@ describe('pie-client-side-controller', () => {
...
     done();
   });
 
+  describe('getLanguages', () => {
+    let langs;
+    beforeEach(() => {
+      langs = controller.getLanguages();
+    });
+
+    it('should return the intersection of the supported languages', function() {
```
Comment (edeustace) [inline test/index-test.js:64]: can we test where `langs` is null, undefined and empty?

File: lib/index.js (hunk 1 above)
Comment (amiklosi) [reply lib/index.js:25]: it might be more lodashy, but it is also not the same unfortunately

File: test/index-test.js (hunk 2 above)
Comment (amiklosi) [reply test/index-test.js:64]: yes, we can!

**Pull request merged**